vips/vip-258/bsctestnet.ts

```diff
@@ -0,0 +1,98 @@
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: COMPTROLLER,
+        signature: "_setAccessControl(address)",
+        params: [ACM],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [COMPTROLLER, "_supportMarket(address)", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [COMPTROLLER, "_setCollateralFactor(address,uint256)", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [COMPTROLLER, "_setLiquidationIncentive(uint256)", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [COMPTROLLER, "_setMarketBorrowCaps(address[],uint256[])", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [COMPTROLLER, "_setMarketSupplyCaps(address[],uint256[])", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [COMPTROLLER, "_setProtocolPaused(bool)", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [COMPTROLLER, "_setActionsPaused(address[],uint8[],bool)", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [COMPTROLLER, "_setForcedLiquidation(address,bool)", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [COMPTROLLER, "_setForcedLiquidation(address,bool)", FAST_TRACK_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [COMPTROLLER, "_setForcedLiquidation(address,bool)", CRITICAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [COMPTROLLER, "_setForcedLiquidationForUser(address,address,bool)", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [COMPTROLLER, "_setForcedLiquidationForUser(address,address,bool)", CRITICAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [COMPTROLLER, "_setForcedLiquidationForUser(address,address,bool)", FAST_TRACK_TIMELOCK],
+      },
```

Comment:
Taking into account [this](https://github.com/VenusProtocol/vips/pull/195/files#diff-dad0819a438f9d7dbf9ec2fd23d357cc54209be597c45c743127a4a1896397cdR182-R186), I would authorize the three timelock contracts to execute `seizeVenus(address[],address)`